crates/quickjs-wasm-rs/src/js_binding/value.rs

```diff
@@ -278,6 +315,12 @@ impl Value {
     }
 }
 
+impl Into<JSValue> for Value {
```

Comment:
Had to tell Clippy to ignore this one. `JSValue` is generated by bindgen and I can't easily add an impl. Additionally, it would lead to a cyclic crate dependency, which is worth avoiding, imo :D Added a comment here as well.